lib/selectivity/rspec.rb

```diff
@@ -37,11 +37,12 @@ def selectivity_unselect(value, *args)
       private
 
       def find_selectivity_input(from, options)
-        find(:div, from, options)
-      rescue Capybara::ElementNotFound
-        label = find('label', { text: from }.merge(options))
-
-        find(:div, "##{label[:for]}", options)
+        label = first(:xpath, ".//label[contains(., '#{from}')]", options)
+        unless lable.nil?
+          find(:div, "##{label[:for]}", options)
+        else
+          first(:xpath, ".//div[contains(., '#{from}')]", options).first('.selectivity-input')
```

Comment:
Sorry for the late response. No, the repository does not have any test suite. I have a separate repository for demo application with simple specs, but it's only uses selenium